spec/rubocop/cop/layout/multiline_method_call_indentation_spec.rb

```diff
@@ -304,6 +304,15 @@ def foo
         RUBY
       end
 
+      context 'target_ruby_version >= 2.5', :ruby25 do
+        it 'accepts blocks passed to class constant' do
```

Comment:
This issue is with key access to hash rather than constant class. For example, the following can be reproduced.

```diff
-            Try[ZeroDivisionError] { 10 / 0 }
+            hash[key] { 10 / 0 }
               .fmap { |x| x * 3 }
```

Can you update the test description and changelog entry?